log.py

```diff
@@ -34,16 +34,16 @@ def format(self, record):
             data['traceback'] = self.formatException(record.exc_info)
         return json.dumps(data)
 
-class UTF8Formatter(logging.Formatter):
-    """Encode all Unicode output to UTF-8 to prevent encoding errors."""
+
+class StringFormatter(logging.Formatter):
+    """Encode all output as a string.
+    
+    In Python 2, this means a UTF-8 bytestring. In Python 3, it means a
+    Unicode string.
+    """
```

Comment:
This is a copy of a file from core, so the same changes need to be made here as in core.